modules/administration-guide/pages/calculating-che-resource-requirements.adoc

```diff
@@ -10,47 +10,63 @@
 
 The {prod-short} Operator, {devworkspace} Controller, and user workspaces consist of a set of pods.
 The pods contribute to the resource consumption in CPU and memory limits and requests.
-Learn how to calculate resources, such as memory and CPU, required to run {prod}.
 
-.Procedure
+To learn how to calculate resources such as memory and CPU that are required to run {prod}, you can use a provided example devfile.
+
+.Prerequisites
 
-. Identify the workspace resource requirements which depend on the devfile that is used for defining the development environment. To illustrate the procedure a specific link:https://github.com/che-incubator/quarkus-api-example/blob/main/devfile.yaml[devfile] will be used for calculating the resource requirements. Identify the workspace components explicitly specified in the `components` section of the devfile.
+* You have downloaded the link:https://github.com/che-incubator/quarkus-api-example/blob/main/devfile.yaml[example devfile].
 +
-** The `tools` component of the devfile defines the following requests and limits:
+IMPORTANT: The previous link is a pointer to material from the upstream community. This material represents the very latest available content and the most recent best practices. These tips have not yet been vetted by Red Hat's QE department, and they have not yet been proven by a wide user group. Please, use this information cautiously. It is best used for educational and 'developmental' purposes rather than 'production' purposes.
```

Comment:
I do not understand why we need this warnign

Reply:
This has been discussed on Slack.